src/test/java/guitests/PlatformExTest.java

```diff
@@ -12,11 +12,6 @@
 
 public class PlatformExTest extends UITest {
 
-    @Override
-    public void launchApp() {
-        FXTestUtils.launchApp(TestUI.class);
```

Comment:
I need to remove override so that it is in test mode; otherwise the test cannot find whether HT main window is the stage or updater is the stage. What do you think?